mne/viz/tests/test_3d.py

```diff
@@ -303,12 +323,13 @@ def test_plot_alignment(tmpdir, renderer, mixed_fwd_cov_evoked):
     info['dig'] = None
     info_cube['chs'][0]['coil_type'] = 9999
     with pytest.raises(RuntimeError, match='coil definition not found'):
-        plot_alignment(info_cube, meg='sensors', surfaces=())
+        _create_meg_coils([info_cube['chs'][0]], acc='normal')
```

Comment:
Why this change from public to private call? Sticking to public interfaces as much as possible in tests is better